features/step_definitions/candidates/registrations/wizard_steps.rb

```diff
@@ -35,13 +35,21 @@
   click_button 'Continue'
 end
 
-Given("I have completed the education form") do
+Given("I have completed the education form (select)") do
   visit path_for 'education', school: @school
   choose 'Graduate or postgraduate'
-  select 'Physics', from: 'If you have or are studying for a degree, tell us about your degree subject'
+  select "Physics", from: "What subject are you studying?"
   click_button 'Continue'
 end
 
+Given("I have completed the education form (type-in)") do
+  visit path_for 'education', school: @school
+  choose 'Graduate or postgraduate'
+  fill_in "What subject are you studying?", with: "Physics"
+  click_button 'Continue'
+  click_button 'Continue' # for reasons unknown, we need to click on this specific button twice in Capybara
```

Comment:
is there some async activity going on here that requires a delay? does `sleep 1` have the same effect? if so, could perhaps solve with some sort of `expect(page).to have_field` shenanigan